spa/src/components/content/contributor-portal/ContributorPortal.tsx

```diff
@@ -53,6 +53,11 @@ const ContributorPortal = ({ revenueProgram }: ContributorPortalProps) => {
     [contact_email, contact_phone, revenueProgram?.contact_email, revenueProgram?.contact_phone]
   );
 
+  const onlyCountryCode = contact_phone.length > 0 && contact_phone.length < 6;
```

Comment:
```suggestion
  const onlyCountryCode = /^\+\d+$/.test(contact_phone);
```

I might have this wrong, but I think this approach is a little more flexible.

Reply:
This doesn't work @nrh-cklimas. `usePhoneInput` has 2 phone strings, `{ phone: string; inputValue: string }`
1. inputValue is the formatted one, +1 (555) 555555 (this is the one displayed in the input)
2. phone is unformatted, +1555555555 (this one is the actual value in `contact_phone`)

Your suggestion works with `inputValue`, but not with `phone` (which is what we use).

Reply:
As discussed on Slack, I was seeing weird behavior with leftover data from the previous PR. I don't think that's going to be a problem here so resolving.